core/lib/tee_verifier/src/lib.rs

```diff
@@ -313,14 +356,11 @@ mod tests {
                 default_validation_computational_gas_limit: 0,
                 chain_id: Default::default(),
             },
-            vec![(H256([1; 32]), vec![0, 1, 2, 3, 4])],
         );
         let tvi = TeeVerifierInput::new(tvi);
-        let serialized = <TeeVerifierInput as StoredObject>::serialize(&tvi)
-            .expect("Failed to serialize TeeVerifierInput.");
+        let serialized = bincode::serialize(&tvi).expect("Failed to serialize TeeVerifierInput.");
         let deserialized: TeeVerifierInput =
-            <TeeVerifierInput as StoredObject>::deserialize(serialized)
-                .expect("Failed to deserialize TeeVerifierInput.");
+            bincode::deserialize(&serialized).expect("Failed to deserialize TeeVerifierInput.");
```

Comment:
Dumb question: What does this test now? Is `bincode` serialization applied to `TeeVerifierInput` elsewhere?

Reply:
We might want to use it for optimization in the future. Bandwidth plays a bigger role here